src/index.js

```diff
@@ -38,6 +38,8 @@ module.exports.GSN = require('./gsn');
 
 module.exports.Token = require('./token');
 
+module.exports.Storage = require('./storage/storage');
```

Comment:
IMO that level of granularity is going way to far, if you go like that we will need module for shop, another for wallet.... althought all of them belongs to the same logic which is an js-sdk for leth features. Therefore I am very much incline to encapsulate every leth logic we are intending to migrate into `./leth` and have:
- `./leth/token.js`
- `./leth/storage.js`
- `./leth/shop.js`
....

Reply:
Ok. But then we also have to move the `gateway` because I followed the current level of granularity. As we discussed on call, once we have time for this refactor: https://github.com/lightstreams-network/lightstreams-js-sdk/issues/23